### ISSUE TITLE
BeWell Android

### ISSUE THREAD

**Issue by bethlash** [ovikariy/bewell#7]
Hi!
The BeWell app currently does not show up in Google Play and the link on the website doesn't work.  Is the app no longer available for Android?

Thanks!

**Comment by ovikariy** [comment]
Yes, at the moment it's not available on Android but I will try to get it back up on the play store soon! Maybe in a week or so.

**Comment by bethlash** [comment]
Thanks for the update!

[Bowling Green State University]
Beth Ash (She/Her/Hers)
Program Manager C2S2

Bowling Green State University
102 Education Bldg
Bowling Green, OH 43403
Office: 419-372-5572



From: Olena Vikariy ***@***.***>
Date: Wednesday, May 1, 2024 at 4:24 PM
To: ovikariy/bewell ***@***.***>
Cc: Beth Ash ***@***.***>, Author ***@***.***>
Subject: [EXTERNAL] Re: [ovikariy/bewell] BeWell Android (Issue #7)

Yes, at the moment it's not available on Android but I will try to get it back up on the play store soon! Maybe in a week or so.
